packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx

```diff
@@ -19,19 +19,19 @@ export const Appearance = () => {
         <InteractionTagPrimary icon={<CalendarMonth />} hasSecondaryAction>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28909&commitId=31afd281a024d86332756a920eb906f9c20e7f96&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - Dark Mode.pressed dismiss.chromium.png|115| Changed|
|InteractionTag Converged.Brand.pressed dismiss.chromium.png|131| Changed|
|InteractionTag Converged.Filled - Dark Mode.pressed dismiss.chromium.png|119| Changed|
|InteractionTag Converged.Outline - Dark Mode.pressed dismiss.chromium.png|4| Changed|
|InteractionTag Converged.Filled.pressed dismiss.chromium.png|80| Changed|
|InteractionTag Converged.Outline.pressed dismiss.chromium.png|72| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>